tt_eager/tt_dnn/op_library/composite/composite_ops.cpp

```diff
@@ -909,6 +909,57 @@ Tensor trunc(const Tensor& input, const MemoryConfig& output_mem_config) {
     return operation::decorate_as_composite(__func__, _trunc)(input, output_mem_config);
 }
 
+Tensor _frac(const Tensor& input, const MemoryConfig& output_mem_config) {
```

Comment:
Similar to this issue: https://github.com/tenstorrent/tt-metal/pull/9365/files
Please add asserts for wormhole b0 arch:
```    
auto arch = input.device()->arch();
TT_FATAL(arch == tt::ARCH::WORMHOLE_B0, "Op is only supported on Wormhole");
```